dockerfiles/Dockerfile.react-native-android-builder

```diff
@@ -0,0 +1,50 @@
+FROM openjdk:8-jdk-stretch
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update
+RUN apt-get install -y gradle jq git-core curl build-essential openssl libssl-dev nodejs
+
+WORKDIR /sdk
+RUN wget https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip -q
+RUN unzip -q sdk-tools-linux-4333796.zip
+
+ENV PATH="${PATH}:/sdk/tools:/sdk/tools/bin"
+
+RUN yes | sdkmanager "platform-tools" "build-tools;28.0.3"
+
+ENV PATH="${PATH}:/sdk/platform-tools"
+ENV ANDROID_HOME="/sdk/"
+
+WORKDIR /app
+
+ENV GRADLE_OPTS="-Dorg.gradle.daemon=false"
+
+COPY test/react-native/features /app/features
+
+ARG REACT_NATIVE_VERSION
+ENV REACT_NATIVE_FIXTURE=${REACT_NATIVE_VERSION}
+
+RUN cp -r /app/features/fixtures/app/ /app/features/fixtures/${REACT_NATIVE_VERSION}/app/
+
+WORKDIR /app/features/fixtures/${REACT_NATIVE_VERSION}
+
+COPY .git .git
+
+RUN rm -f ~/.npmrc
+ARG REG_BASIC_CREDENTIAL
+ARG REG_NPM_EMAIL
+RUN echo "_auth=$REG_BASIC_CREDENTIAL" >> ~/.npmrc
+RUN echo "email=$REG_NPM_EMAIL" >> ~/.npmrc
+RUN echo "always-auth=true" >> ~/.npmrc
+
+ARG REGISTRY_URL
+
+RUN npm install --registry $REGISTRY_URL
+RUN npm install @bugsnag/react-native@$(git rev-parse --short HEAD) --registry $REGISTRY_URL
+
+RUN ./node_modules/.bin/react-native link
```

Comment:
This only needs to happen on RN pre 0.60. I think it will just log a warning in 0.60+ but they may remove it completely at some point.

Reply:
I agree, considering we only have a single fixture that requires this I'll look at adding a conditional to only run on that version (0.55).